src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -51,13 +79,17 @@ protected function create_post( array $record ) {
 		return $id;
 	}
 
-	private function get_event_start_date( array $record ) {
+	private function get_event_start_date( array $record, $date_only = false ) {
 		$start_date = $this->get_value_by_key( $record, 'event_start_date' );
 		$start_time = $this->get_value_by_key( $record, 'event_start_time' );
-		if ( ! empty( $start_time ) ) {
+
+		if ( ! $date_only && ! empty( $start_time ) ) {
 			$start_date .= ' ' . $start_time;
 		}
-		$start_date = date( 'Y-m-d H:i:s', strtotime( $start_date ) );
+
+		$start_date = $date_only
+			? date( 'Y-m-d', strtotime( $start_date ) )
```

Comment:
We should use the `Date_Utils::DBDATEFORMAT` and `Date_Utils::DBDATETIMEFORMAT` also I think we could make this ternary a real if conditional.